src/crystal/system/win32/file_descriptor.cr

```diff
@@ -9,7 +9,12 @@ module Crystal::System::FileDescriptor
 
   # Platform-specific type to represent a file descriptor handle to the operating
   # system.
-  alias Handle = ::LibC::Int
+  # NOTE: this should really be `LibC::HANDLE`
```

Comment:
**question:** Why is it not then? I'm not sure which it should really be. And I'm wondering if it even matters much. But probably, `UIntPtrT` is just fine.
Anyway, if leaving such a note, we should give a reason.